corefxlab.sln

```diff
@@ -112,7 +112,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Numerics.Experimenta
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Data.DataFrame", "src\Microsoft.Data\Microsoft.Data.DataFrame.csproj", "{AD22AAD4-FCB0-4DB4-BC38-AB6ACD153899}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Data.DataFrame.Tests", "tests\Microsoft.Data.Tests\Microsoft.Data.DataFrame.Tests.csproj", "{485FC567-4AEC-4335-B767-283173F64C42}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Data.DataFrame.Tests", "tests\Microsoft.Data.DataFrame.Tests\Microsoft.Data.DataFrame.Tests.csproj", "{9C354CA7-5CDF-44F9-A884-E5840736DF36}"
```

Comment:
Why the change in GUID here?

`FAE04EC0-301F-11D3-BF4B-00C04F79EFBC` is the "old style" project GUID, I believe. This should stay as `9A19103F-16F7-4668-BE54-9A1E7A4F7556`

Reply:
Hmm, I wonder if this happened when I rebased on master cherry-picked a commit. Any idea on how I should go about fixing this? I'm not super good with csproj/sln stuff. If I revert the changes to this file alone, would it break anything else?

Reply:
Just hand edit this sln to revert this one guid (FAE.. to 9A19..). Reopen in VS, if you then hit save on it and VS changes it back - that is odd...